vars/createGithubIssue.groovy

```diff
@@ -27,7 +27,10 @@ def call(Map args = [:]){
             if (failedComponents.contains(component.name)) {
                 println("Component ${component.name} failed, creating github issue")
                 compIndex = failedComponents.indexOf(component.name)
-                create_issue(component.name, component.repository, currentVersion, failureMessages[compIndex])
+                if (component.name == "OpenSearch"){
```

Comment:
Hey @rishabh6788 thanks for the PR, this will only create issue for a component `OpenSearch`, we might need for other components as well, the ask is this (correct me if i'm wrong), if a component would not want to create an issue upon failure that specific component has to be excluded, the component can `OpenSearch` or any other plugin as well.

Reply:
@prudhvigodithi I dont think we should do that. A component is added to the manifest only when they are ready. IF its failing to build, it is good to have an issue open irrespective of wanting it or not.

Reply:
I'm fine with it, there was an ask from plugin team if there is a way to exclude from creating a ticket, but this PR does not solve that, it only creates issue for `OpenSearch` which is what I was saying @gaiksaya 
@bbarani

Reply:
I believe we told them that this wont be happening since we want the issues to be created if the build failed. If the component is not ready just dont add them in the manifest then until you are.

Reply:
Again as I said, i'm ok that way, just conveying what I heard and reviewing the PR in accordingly ;)

Reply:
Wondering if we should keep the failure for some another component. Did OpenSearch ever fail? I saw security failing a bunch of times

Reply:
@gaiksaya @prudhvigodithi as discussed removed the condition and enabled issue creation for all components.